### PR TITLE
As a user I want to have my real name everywhere

### PR DESCRIPTION
**Pull request by VitalikMatvieiev**
…recording locally

### CODE REVIEW

File: backend/Recodo.BLL/MappingProfiles/UserProfile.cs [hunk 1]
```diff
@@ -8,8 +8,8 @@ public class UserProfile : Profile
     {
         public UserProfile()
         {
-            CreateMap<NewUserDTO, User>();
-            CreateMap<User, UserDTO>();
+            CreateMap<NewUserDTO, User>().ReverseMap();
```
Comment (Jyuart) [inline backend/Recodo.BLL/MappingProfiles/UserProfile.cs]: Why do you need reverse mapping for NewUserDto containing user password?

File: frontend/src/app/services/resource.service.ts [hunk 2]
```diff
@@ -58,4 +59,10 @@ export abstract class ResourceService<T> {
     return throwError(() => error);
   }
 
+  public getFullRequest<TRequest>(url: string, httpParams?: HttpParams): Observable<HttpResponse<TRequest>> {
```
Comment (Jyuart) [inline frontend/src/app/services/resource.service.ts:62]: Why do you need this type of request?

File: frontend/src/app/services/user.service.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    providedIn: 'root',
+  })
+
+export class UserService {
+    private currentUser: UserDto = {} as UserDto;
+    private unsubscribe$ = new Subject<void>();
+
+    constructor(private registrationservice: RegistrationService) {}
+
+    public getAutorithedUser() {
```
Comment (Jyuart) [inline frontend/src/app/services/user.service.ts]: It's never used

File: frontend/src/app/modules/workspace/navbar/navbar.component.html [hunk 4]
```diff
@@ -7,9 +7,17 @@
...
\ No newline at end of file
+    <mat-sidenav *ngIf="isUserPanelOpen; else closed" mode="over" opened position="end">
+        <img src="../../assets/icons/test-user-logo.png" alt="UserAvatar" class="user-image">
+        <p>{{this.user.workspaceName}}</p>
+        <button mat-button class="workspace-button" (click)="toggleUserPersonalPanel()">Close</button>
+    </mat-sidenav>
+    <ng-template #closed>
+        <mat-sidenav mode="over" position="end"></mat-sidenav>
+    </ng-template>
+</mat-toolbar>
```
Comment (Jyuart) [inline frontend/src/app/modules/workspace/navbar/navbar.component.html]: It's not a part of this ticket. Not sure we need it

**Comment by Jyuart** [comment]
And please fix all linter warnings:
- `ng lint --fix`
- manually
- using prettier

**Comment by Jyuart** [comment]
And add some kind of caching in localStorage to not ask backend each time for this info

File: backend/Recodo.BLL/MappingProfiles/UserProfile.cs (hunk 1 above)
Comment (VitalikMatvieiev) [reply backend/Recodo.BLL/MappingProfiles/UserProfile.cs]: agree fixed it

File: frontend/src/app/services/resource.service.ts (hunk 2 above)
Comment (VitalikMatvieiev) [reply frontend/src/app/services/resource.service.ts:62]: with this method, current class stay more flexible

File: frontend/src/app/services/user.service.ts (hunk 3 above)
Comment (VitalikMatvieiev) [reply frontend/src/app/services/user.service.ts]: fixed

File: frontend/src/app/modules/workspace/navbar/navbar.component.html (hunk 4 above)
Comment (VitalikMatvieiev) [reply frontend/src/app/modules/workspace/navbar/navbar.component.html]: deleted

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by IvanitskyiVolodymyr — DISMISSED** [review]

**Review by IvanitskyiVolodymyr — APPROVED** [review]

**Review by Jyuart — APPROVED** [review]

**Pull request merged**